tests/tiup/conf/task_pessimistic.yaml

```diff
@@ -4,6 +4,7 @@ task-mode: all
 is-sharding: true
 shard-mode: pessimistic
 enable-heartbeat: true
+case-sensitive: true
```

Comment:
I forget some discussion, what's the behaviour when upstream is sensitive and config is not sensitive?

Reply:
The behaviour is the same as the current logic, this PR just change the db/table name in db/etcd from lower case to origin case.